pkg/virtctl/console/console.go

```diff
@@ -177,6 +178,11 @@ func (c *Console) Run(cmd *cobra.Command, args []string) error {
 	terminal.Restore(int(os.Stdin.Fd()), state)
 
 	if err != nil {
+		if strings.Contains(err.Error(), "abnormal closure") {
```

Comment:
A more idiomatic way to check for errors is to treat them as types.
See for example what we do here https://github.com/kubevirt/kubevirt/blob/0acbdb417a0f7721fca9b73bd4be97391bd0ed54/staging/src/kubevirt.io/client-go/kubecli/vmi.go#L121

For more info, see for example https://medium.com/@sebdah/go-best-practices-error-handling-2d15e1f0c5ee 

So the if would be something like
```
	if e, ok := err.(*websocket.CloseError); ok && e.Code == websocket.CloseAbnormalClosure {

Reply:
Oh, interesting! Thanks for that!

Reply:
Ah yes, nice! I always forget about this, because we don't use custom error types ourselves that much.